packages/protocol/contracts/governance/EpochRewards.sol

```diff
@@ -362,7 +369,7 @@ contract EpochRewards is Ownable, Initializable, UsingPrecompiles, UsingRegistry
    *   voting Gold fraction.
    * @dev Only called directly by the protocol.
    */
-  function updateTargetVotingYield() external {
+  function updateTargetVotingYield() external onlyWhenNotFrozen {
```

Comment:
For robustness, `calculateTargetEpochPaymentAndRewards` should return `0, 0` when frozen as no rewards should be distributed.